packages/blocks/src/surface-block/elements/group/group-element.ts

```diff
@@ -0,0 +1,217 @@
+import { assertExists } from '@blocksuite/global/utils';
+
+import type { RoughCanvas } from '../../rough/canvas.js';
+import { Bound } from '../../utils/bound.js';
+import { linePolygonIntersects } from '../../utils/math-utils.js';
+import type { PointLocation } from '../../utils/point-location.js';
+import type { IVec } from '../../utils/vec.js';
+import type { HitTestOptions } from '../edgeless-element.js';
+import { SurfaceElement } from '../surface-element.js';
+import {
+  getFontString,
+  getLineHeight,
+  getLineWidth,
+  truncateTextByWidth,
+} from '../text/utils.js';
+import type { IGroup, IGroupLocalRecord } from './types.js';
+
+export class GroupElement extends SurfaceElement<IGroup, IGroupLocalRecord> {
+  private _cacheChildren: string[] = [];
```

Comment:
Should use `cachedChildren` for naming. Beware that "cached" is adjective but "cache" is a verb. Don't be too careless about basic grammar.